GenericSerialDeviceRemoteControl/Testing/CMakeLists.txt

```diff
@@ -0,0 +1 @@
+add_subdirectory(Python)
```

Comment:
it's a text file, so it is good practice to terminate it with a newline

Reply:
Remove the empty GenericSerialDeviceRemoteControl/Testing/Python/CMakeLists.txt file